src/database/adatabase.cpp

```diff
@@ -18,6 +18,7 @@
 #include "adatabase.h"
 #include "src/testing/adebug.h"
 
+#define DATABASE_VERSION 15
```

Comment:
it should be in the .h file. The point is that its going to be #included to get the value. Right now its... not good